dashboards/README.md

```
@@ -0,0 +1,186 @@
+Metabase infrastructure deployed through terraform with multi-tenant analytics.
+
+## Quick Start
+
+### Prerequisites
+
+- Docker and Docker Compose installed
+- BigQuery service account key saved to `./secrets/bigquerykey.json` (or customize path via `bigquery_service_account_key_path` in terraform.tfvars)
+
+### Setup Steps
+
+**1. Start Metabase containers**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)